core/foundation/inc/ROOT/StringUtils.hxx

```diff
@@ -40,6 +40,8 @@ std::string Join(const std::string &sep, StringCollection_t &&strings)
                           [&sep](auto const &a, auto const &b) { return a + sep + b; });
 }
 
+std::string Round(const double value, const double error, const unsigned short cutoff = 1, const std::string_view delim = "#pm");
```

Comment:
I think `string_view` should not be passed as `const` (neither should `double` and `unsigned short`) as it's a value type

Reply:
Is it because of readability?

I prefer having const, since it will throw a compiler error if by accident one developer in the futures tries to modify any of these variables within the code.

Reply:
In the case of primitives it's mostly for consistency with the rest of the codebase (it's added noise that gives no benefit in real cases - I've almost never seen a case of someone accidentally modifying a parameter when it wasn't intended); 
additionally, as a general rule, value types should not be passed (or returned) as `const` in order to not hinder move opportunities by the compiler.

Reply:
While we're at it, I would personally suggest `std::uint16_t` over `unsigned short` (or just `int`, since it's not really important that the number is a short imho)